hack/demo/catalogd-demo-script.sh

```diff
@@ -36,4 +36,3 @@ curl -k https://localhost:8081/catalogs/operatorhubio/api/v1/all | jq -s '.[] |
 curl -k https://localhost:8081/catalogs/operatorhubio/api/v1/all | jq -s '.[] | select(.schema == "olm.channel") | select(.package == "wavefront") | .name'
 # check what bundles are included in the wavefront package
 curl -k https://localhost:8081/catalogs/operatorhubio/api/v1/all | jq -s '.[] | select(.schema == "olm.bundle") | select(.package == "wavefront") | .name'
-
```

Comment:
👍 It is OK, but we might want to have a complete demo instead of keep just for catalogd.
However, until there, why not? Who knows, we change the intention as well.

Reply:
The intention here is that I'm creating demo scripts for the own/single namespace install mode feature, which I'll add to the demos directory. I'll probably add a Makefile at some point.